### PR TITLE
HW-21 Glovo spring appp DB storage

### PR DESCRIPTION
**Pull request by IvashchenkoIvanIgorevich**

### CODE REVIEW

**Review by JzomitiZumatti — APPROVED** [review]
I am not sure about the correctness of the choice a relationship between two entities, but everything else on my opinion is great.

**Comment by NatSjm** [comment]
Good job!

**Review by ExQA — APPROVED** [review]
Nice work!

File: glovo/src/main/java/org/example/glovo/service/OrderService.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        orderRepository.save(foundOrder);
+
+        return true;
+    }
+
+    public Order delete(Long id) throws OrderNotFoundException {
+        if (id == null) {
+            throw new IllegalArgumentException("Order id is a required argument");
+        }
```
Comment (shaitanaleksander) [inline glovo/src/main/java/org/example/glovo/service/OrderService.java:117]: may be better to go with primitive  long instead of Long

**Review by shaitanaleksander — APPROVED** [review]

File: glovo/src/main/java/org/example/glovo/service/OrderService.java (hunk 1 above)
Comment (IvashchenkoIvanIgorevich) [reply glovo/src/main/java/org/example/glovo/service/OrderService.java:117]: agree!

**Pull request merged**